src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -17,7 +17,10 @@
 import os
 import shutil
 
-from libFuzzer import constants
+from . import engine_common
```

Comment:
Please use python 2 style imports for now, rather than relative imports. We will be transitioning to Py3, and currently this might cause an issue (see also revert https://github.com/google/clusterfuzz/commit/559c76b4e04c55e940309c2fb7383037ff8c3e60). basically we call launcher.py in a new shell and it can cause `ValueError: Attempted relative import in non-package`

just do import engine_common
from libFuzzer import constants.

Reply:
ah, sorry, must've rebased at the wrong time. fixed